### PR TITLE
fixed mart code & materialization

### PR DESCRIPTION
**Pull request by maham-creates**

### CODE REVIEW

File: transform/dbt_project.yml [hunk 1]
```diff
@@ -31,5 +23,14 @@ clean-targets:         # directories to be removed by `dbt clean`
 # files using the `{{ config(...) }}` macro.
 models:
   github_stars:
-    # Config indicated by + and applies to all files under models/example/
+  
+    staging:
+      +enabled: true
```
Comment (edsioufi) [inline transform/dbt_project.yml]: Minor: Let's consider removing enabled here.

**Review by edsioufi — APPROVED** [review]
Great job! 🚀 

Approved with a minor comment that I'll let you fix optionally. If you decide to fix it, just add a commit to this PR's branch and ping me for re-review.

File: transform/dbt_project.yml (hunk 1 above)
Comment (maham-creates) [reply transform/dbt_project.yml]: made changes

**Review by edsioufi — APPROVED** [review]
Looks great! 👍

**Pull request merged**